example/hello_ll.c

```diff
@@ -20,6 +20,15 @@
 
 #define FUSE_USE_VERSION 34
 
+#ifdef IO_UDS_STREAMS
```

Comment:
Can we leave hello_ll.c as-is (it's meant to be the simplest possible example) and instead add a new example that demonstrates how to use a custom fd? I know you wrapped it in an ifdef, but that doesn't help the reader all that much...

Reply:
That's fair. Is it ok if I make a new example that copies a lot of the code from hello_ll.c? Since I think extracting common code between hello_ll.c and this new example I will create to a separate file also makes the hello_ll.c example less clear

Reply:
Yes, having code duplication between the two examples is fine.